packages/config-cli/README.md

```diff
@@ -0,0 +1,18 @@
+# Basemaps Config CLI
+
+This package is to control the configuration in the LINZ basemaps product.
+
+## Usage -- Screenshots
+
+Dump the screenshots from basemaps production
+
+```bash
+./bmc.js screenshot
```

Comment:
I dont see a `./bmc.js` do these commands work?

Reply:
Yes, that's how does this work for me without install the package.

Reply:
ahh was meaning in the `package.json` this file would not have be published. I expanded on it in my next comment